docs/src/time_integration.md

```diff
@@ -14,3 +14,13 @@ Interesting classes of time integration schemes are
   of stages, e.g. to allow for interpolation (dense output), root-finding for continuous callbacks,
   and error-based time step control. In general, you often should not need to worry about this if you
   use Trixi.
+
+Some common options for `solve` are the following. Further documentation can be found
+in the [SciML docs](https://diffeq.sciml.ai/v6.8/basics/common_solver_opts/).
+- If you use a fixed time step method like `CarpenterKennedy2N54`, you need to pass
+  a time step as `dt=...`. If you use a `CFLCallback`, the value passed as `dt=...`
```

Comment:
While debugging the Alfven test, this caused us a lot of headaches. Thus I am wondering if we should just hard-code a non-sensible (or default) value here in the elixirs, and maybe add a single elixir where this is actually used? I don't want to remove this feature in general (since someone else might rely on it), but OTOH it feels weird to always compute stuff we never need. What do you think?

Reply:
That's okay for me (as long as we keep it in at least one elixir to document/test it).

Reply:
Done in 1aa15cd. `examples/1d/elixir_advection_amr_nonperiodic.jl` is the only example keeping the old version.

Reply:
:+1: